drivers/soundwire/qcom.c

```diff
@@ -821,8 +821,10 @@ static int qcom_swrm_probe(struct platform_device *pdev)
 		goto err_clk;
 	}
 
-	ret = sdw_add_bus_master(&ctrl->bus);
-	if (ret) {
+	ctrl->md = sdw_add_bus_master(&ctrl->bus, dev, dev->fwnode,
+				      NULL, NULL);
```

Comment:
this breaks bisection, you cannot do this separately from actually changing the function